Makefile

```diff
@@ -1,82 +1,94 @@
-CFLAGS=-g 
+CXX=g++
+CXXFLAGS=-g
+LDLIBS=-lreadline -lwebsockets -lpthread -lrt -lssl -lcrypto
+SERVER=server
+EMULATOR=msp430-emu
 
-all: MSP430 SERVER
+.PHONY: all test
```

Comment:
also `clean` should be there, since this rule does not produce any file

Reply:
Ack